source/declarations.tex

```diff
@@ -495,9 +495,9 @@
 
 \pnum
 \begin{note}
-The \tcode{mutable} specifier on a class data member nullifies a
+The \keyword{mutable} specifier on a class data member nullifies a
```

Comment:
Hm, I don't think I should be using \keyword in non-normative text, since it adds to the index. @jensmaurer can you confirm?

Reply:
What's wrong with using it in non-normative text (and thus getting an index entry)? If we ever color the standard, we definitely want keywords to show up like keywords everywhere. If we feel the index becomes less usable, we should find a way to highlight the "definition" of a keyword.

Reply:
Good point.